src/Commands/RunRepeater.ts

```diff
@@ -164,7 +162,7 @@ export class RunRepeater implements CommandModule {
               exchange: 'EventBus',
               clientQueue: `agent:${args.id as string}`,
               connectTimeout: 10000,
-              url: args.bus as string,
+              url: Helpers.getClusterUrls(args).bus as string,
```

Comment:
Well, why just don't assign `bus` and `api` to `args` in `middleware`

e.g.

```ts
.middleware((args) => {
    if (args.cluster) {
       const { bus, api } = Helpers.getClusterUrls(args); 
       args.bus =  bus;
       // ...
    }
  }, true)
```

Reply:
do you mean make it in CLIBuilder?